.github/workflows/ci.yml

```diff
@@ -27,7 +27,30 @@ jobs:
       with:
         name: linux-test-reports
         path: build/reports
-      if: success() || failure()
+      if: always()
+    - name: Run integration tests
+      run: |
+        dir=~/.java/.userPrefs/jetbrains/_\!\(\!\!cg\"p\!\(\}\!\}@\"j\!\(k\!\|w\"w\!\'8\!b\!\"p\!\':\!e@=\=
+        mkdir -p "$dir"
+        cp .github/workflows/prefs.xml $dir
+        mkdir -p ~/.local/share/JetBrains/consentOptions
+        cp .github/workflows/accepted ~/.local/share/JetBrains/consentOptions
+        export DISPLAY=:99.0
+        Xvfb -ac :99 -screen 0 1920x1080x16 &
+        ./gradlew runIdeForUiTests -PideaVersion=2020.1 &
+        ./gradlew integrationTest
+    - uses: actions/upload-artifact@v1
+      with:
+        name: ${{ runner.os }}-integration-test-reports
+        path: build/reports/tests/integrationTest
+      if: always()
+    - name: Archiving screenshots
+      uses: actions/upload-artifact@v2
+      with: 
+        name: screenshots
+        path: |
+          build/screenshots/*
+        retention-days: 5 
```

Comment:
Workflow runs have their own retention time why is this needed ?

Reply:
I have copied this lines of code from @odockal. We came to the conclusion that 5 days is accurate period of time to keep the files. If you think I should delete it, it is ok for me.

**Copied from:**
(https://github.com/redhat-developer/vscode-server-connector/blob/master/.github/workflows/gh-actions.yml#L36)

Reply:
@jeffmaury looks like this is the last issue - do you want me to delete the 'retention-days: 5 ' line or we just leave it as it is (and maybe delete is later)?

Reply:
Remove it

Reply:
Removed.